common/config-parser/src/types/spec.rs

```diff
@@ -258,4 +264,11 @@ impl From<HardforkInput> for HardforkInfoInner {
 #[derive(Clone, Debug, Serialize, Deserialize, Copy, ValueEnum, EnumIter, PartialEq, Eq, Hash)]
 pub enum HardforkName {
     None = 0b0,
+    Andromeda = 0b1,
+}
+
+impl HardforkName {
+    pub fn all() -> u64 {
+        HardforkName::Andromeda as u64
+    }
 }
```

Comment:
In the future, every hardfork implementation comes out, this function should be updated to
accommodate new one, right?

Is it better to iterate the HardforkName and generate HardforkName::all?